pkg/proxy/iptables/proxier.go

```diff
@@ -1515,7 +1515,13 @@ func (proxier *Proxier) syncProxyRules() {
 	klog.V(5).InfoS("Restoring iptables", "rules", proxier.iptablesData.Bytes())
 	err = proxier.iptables.RestoreAll(proxier.iptablesData.Bytes(), utiliptables.NoFlushTables, utiliptables.RestoreCounters)
 	if err != nil {
-		klog.ErrorS(err, "Failed to execute iptables-restore")
+		parseError, pErr := utiliptables.ParseErrorByRestore(err)
+		if pErr != nil {
```

Comment:
if we don't care pErr, ParseErrorByRestore could return (ParseError, bool)

Reply:
yeah, I agree with you